gulpfile.js

```diff
@@ -249,11 +256,11 @@ gulp.task('index', ['jsHint', 'scripts'], function() {
 
 // start local express server
 gulp.task('serve', function() {
-  express()
+  server = express()
     .use(!build ? connectLr() : function(){})
     .use(express.static(targetDir))
     .listen(port);
-  open('http://localhost:' + port + '/');
+  //open('http://localhost:' + port + '/');
```

Comment:
Also, is it necessary to comment this out?
I usually like to just type gulp and it starts the browser window for me